md_api/src/main/java/com/wipro/ats/bdre/md/api/TermDeploy.java

```diff
@@ -42,6 +37,10 @@ public TermDeploy() {
     @Autowired
     DeployDAO deployDAO;
 
+    public TermDeploy() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
     public TermDeployInfo execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)